public/app/features/alerting/unified/RuleViewer.tsx

```diff
@@ -17,10 +20,51 @@ type RuleViewerProps = GrafanaRouteComponentProps<{
 
 const newAlertDetailView = Boolean(config.featureToggles.alertingDetailsViewV2) === true;
```

Comment:
I know this is not related to this PR, but is this comparison necessary?( `Boolean(config.featureToggles.alertingDetailsViewV2)` would be enough in here without comparing to true), 
 Or maybe is for giving more clarity?

Reply:
Yea in this case it's done for readability even though it's definitely not necessary :)